app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -457,11 +457,12 @@ class NimbusReadyForReviewSerializer(
     NimbusStatusRestrictionMixin, serializers.ModelSerializer
 ):
     public_description = serializers.CharField(required=True)
-    proposed_duration = serializers.IntegerField(required=True)
-    proposed_enrollment = serializers.IntegerField(required=True)
+    proposed_duration = serializers.IntegerField(required=True, min_value=1)
+    proposed_enrollment = serializers.IntegerField(required=True, min_value=1)
     population_percent = serializers.DecimalField(
```

Comment:
Not 100% sure, but I don't *think* we actually want to accept values of zero here?